tests/test_module.py

```diff
@@ -1,5 +1,20 @@
-#This file is meant to test the module to ensure it works
+"""
+This file is meant to test the module to ensure it works
+"""
+
+from 2016_summer_XPD.roi.roi_definer import ROI
```

Comment:
I think it is time for us to touch ``setup.py`` now, so you don't have to use weird import directory